airbyte-cdk/python/airbyte_cdk/entrypoint.py

```diff
@@ -44,15 +44,14 @@ def __init__(self, source: Source):
         self.source = source
         self.logger = logging.getLogger(f"airbyte.{getattr(source, 'name', '')}")
 
-    @staticmethod
-    def parse_args(args: List[str]) -> argparse.Namespace:
-        # set up parent parsers
-        parent_parser = argparse.ArgumentParser(add_help=False)
-        parent_parser.add_argument("--debug", action="store_true", help="enables detailed debug logs related to the sync")
-        main_parser = argparse.ArgumentParser()
-        subparsers = main_parser.add_subparsers(title="commands", dest="command")
-
-        # spec
+    @click.command()
```

Comment:
could you refactor this code so that there are four methods: spec, check, discover, and read, each a separate click method?